src/public/admin/alerts/styles.css

```
@@ -0,0 +1,294 @@
+}
+
+.ri-context {
+  margin-top: 0.75rem;
+  border: 1px solid rgba(167, 170, 225, 0.2);
+  background: rgba(255, 255, 255, 0.03);
+  border-radius: 10px;
+  padding: 0.65rem;
+  font-family: 'Courier New', monospace;
+  font-size: 0.72rem;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)